public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataTransformationsTab.test.tsx

```diff
@@ -60,11 +60,12 @@ describe('PanelDataTransformationsModel', () => {
   });
 });
 
-describe('PanelDataTransformationsTab', () => {
+describe.skip('PanelDataTransformationsTab', () => {
```

Comment:
Skipping these tests that relies on the drawer. The drawer still expanding after `act` seems to be causing flakyness